Alloy/MTLTexture+Extensions.swift

```diff
@@ -125,6 +125,49 @@ public extension MTLTexture {
     }
 }
 
+public extension MTLTexture {
+
+    func pixelBuffer() throws -> CVPixelBuffer {
+        let error = MetalError.MTLTextureSerializationError
```

Comment:
this is really strange choice, let's not create an error until we really need it

Reply:
can CVPixelBuffer conversion really be considered as serialization?